citizenlab.config.json

```diff
@@ -9,7 +9,7 @@
     "free/frontend": true,
     "free/onboarding": true,
     "free/polls": true,
-    "free/project_permissions": true,
```

Comment:
Project permissions should stay for now (as long as the backend as not been refactored).